conda/recipes/cucim/meta.yaml

```diff
@@ -20,23 +20,25 @@ build:
 requirements:
   build:
     - cmake >=3.18.0
+    - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    - sysroot_linux-64 2.17
   host:
     - cudatoolkit {{ cuda_version }}.*
     - python {{ python_version }}.*
     - libcucim {{ version }}.*
     - click
-    - conda-forge/label/cupy_rc::cupy=9
-    - numpy
+    - cupy 9.*
+    - numpy >=1.17.0
```

Comment:
Was needed as NumPy 1.16 was being pulled in, which is incompatible with RAPIDS and CuPy 9 and caused solver conflicts. Forcing 1.17 fixed the issue